### DISCUSSION TITLE
Pul shark

### DISCUSSION THREAD

**Discussion by Pflasterlix** [community/community#64872] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

n GitHub, when you pull a request successfully, you receive a shark icon as a success indicator. Why?

**Comment by agetbase** [comment]
In the **repository** at https://github.com/Schweinepriester/github-profile-achievements, you can find detailed information about **`GitHub Profile Achievements`**. If your `PR(Pull Request)` has been successfully merged twice or more, you will earn the **`Pull Shark`** badge.

**Comment by mdnurulmomen** [comment]
The Pull Shark achievement is ranked by how many pull requests you've merged; x1 signifies that you've merged at least 2 pull requests, x2 signifies that you've merged at least 16 pull requests, and x3 signifies that you've merged at least 128 pull requests.

For more [GitHub Achievements](https://github.com/drknzz/GitHub-Achievements)

**Comment by 0dayhunter** [comment]
The shark icon that you're referring to is actually not an official feature or indicator provided by GitHub. As of my last knowledge update in September 2021, GitHub does not use a shark icon as a success indicator for pull requests. GitHub primarily uses checkmarks, green "merge" buttons, and other UI elements to indicate successful pull requests.

It's possible that what you're seeing could be a custom browser extension, user script, or third-party tool that you've installed. These tools can modify the appearance and behavior of websites, including GitHub, to add custom icons or features.

If you're seeing a shark icon in relation to pull requests on GitHub, I recommend investigating any browser extensions or user scripts you might have installed. Keep in mind that any customizations made through third-party tools might not be officially supported by GitHub and could change or behave unexpectedly.
